app/controllers/Subscriptions.scala

```diff
@@ -57,4 +57,15 @@ class Subscriptions(
     }
   }
 
+  def digitalGeoRedirect: Action[AnyContent] = GeoTargetedCachedAction() { implicit request =>
+    val redirectUrl = request.fastlyCountry match {
+      case Some(UK) => "/uk/subscribe/digital"
+      case Some(US) => "/us/subscribe/digital"
+      case Some(RestOfTheWorld) => "/int/subscribe/digital"
+      case _ => "https://subscribe.theguardian.com/digital"
```

Comment:
We are going to be sending Canada, NZ and all of Europe to the default case, is that the desired behaviour?

Reply:
Yes, we don't have a digital page for them yet.

Reply:
👍